applicationLibrary/molecularDynamics/molecularDynamicsLibrary/LJFunctor.h

```diff
@@ -820,8 +820,8 @@ class LJFunctor : public autopas::Functor<Particle, LJFunctor<Particle, applyShi
       return (static_cast<double>(numKernelCallsNoN3Acc) + static_cast<double>(numKernelCallsN3Acc)) /
              (static_cast<double>(numDistCallsAcc));
     } else {
-      autopas::utils::ExceptionHandler::exception("LJFunctor::getHitRate called without countFLOPs enabled!");
-      return 0;
+      // This is needed because this function still gets called with FLOP logging disabled, just nothing is done with it
+      return std::numeric_limits<size_t>::quiet_NaN();
```

Comment:
Is it on purpose that you chose `quiet_NaN()` here and `max()` in line 804?

Reply:
Yes. `quiet_NaN()` is more appropriate for an invalid number, but `size_t` returns 0 for this, which is not desired. (Actually, it is a mistake that this is `size_t` here, and is changed to `double` in a824ccee)